docs/build/p2p-node/index.md

```diff
@@ -1,25 +1,19 @@
-contributors: <Dennis Wellmann:dennis@centrifuge.io>
+title: Centrifuge POD (P2P Node)
+contributors: <Cosmin Damian:cosmin@centrifuge.io>
 ---
 
 ## Introduction
 
-### Centrifuge P2P Node
+### Centrifuge POD (P2P Node)
 
 The Centrifuge P2P network is built to support a new generation of applications for the financial supply chain. Centrifuge provides users with the ability to remove intermediaries and create financial business documents as Non-Fungible Tokens (NFTs) that have long-term verifiability, are censorship resistant, and are stored and processed in a decentralized fashion.
 
 The Centrifuge P2P node provides a method to create, exchange, and use the data that exists in the financial supply chain. It creates transparent and shareable relationships between interacting companies. Data owners can selectively share the information with their business partners or other users of the network. Centrifuge provides a censorship resistant way to verify the authenticity of data that is transacted through and stored in it. This creates the foundation for data ownership, privacy, and transparency throughout the financial supply chain and also allows third parties to offer additional services, such as instant and decentralized financing of invoices and orders, trade credit insurance and financing supply chains multiple levels deep.
 
-The underlying Centrifuge protocol has a two layered approach. It is built on Ethereum & Substrate which allows businesses to transact freely on a single verifiable source of truth. The public blockchains are used for business identities, committing document status and minting business NFTs. In addition, a peer to peer network enables private, off-chain transactions and is used to send business documents back and forth.
-
-### Ethereum
-
-Ethereum is a decentralized platform that runs smart contracts. Functionality deployed on Ethereum acts as the primary and trusted entry point and anchor for the other Centrifuge components. The censorship resistance and decentralized nature of the platform provides trustless lookups for users. Ethereum allows users to publish identity and document information through our Centrifuge Smart Contracts without relying on any third party. Furthermore, Ethereum’s architecture provides a well-tested system for the issuance of tokens to incentivize honest participation within the OS, issuance of tokens directly related to business documents, as well as decentralized governance layers of the OS at large.
-
-For more information, see the [Ethereum Project](https://www.ethereum.org/).
+The underlying Centrifuge protocol has a two layered approach. It is built on Substrate which allows businesses to transact freely on a single verifiable source of truth. The public blockchain is used for business identities, committing document status and minting business NFTs. In addition, a peer to peer network enables private, off-chain transactions and is used to send business documents back and forth.
```

Comment:
I am not sure, but the `off-chain transactions` part is confusing for me. Is this referring to the ability to collaborate on a document and sign those document?

Reply:
Yes, we could change that to something like:

```
In addition, a peer to peer network enables the exchange of business documents.
```

Reply:
I like that, I would add:
```
In addition, a peer to peer network enables the exchange of business documents in a private and verifiable way.
```